### PR TITLE
[editor] Fix crash while loading feature created on deleted mwm.

### PR DESCRIPTION
**Pull request by mgsergio**
@ygorshenin @syershov

### CODE REVIEW

**Comment by syershov** [comment]
LGTM

File: indexer/osm_editor.cpp [hunk 1]
```diff
@@ -195,6 +202,12 @@ void Editor::LoadMapEdits()
         {
           XMLFeature const xml(nodeOrWay.node());
 
+          if (!m_isMwmForPointPresentFn(GetSomeFeaturePoint(xml)))
```
Comment (ygorshenin) [inline indexer/osm_editor.cpp]: Probably you need to use mwmId.IsAlive() here instead of this functor.

**Comment by ygorshenin** [comment]
Тесты?

**Comment by ygorshenin** [comment]
TODO поправить ситуацию со сменой именования?

**Comment by ygorshenin** [comment]
А, ОК. LGTM

**Pull request merged**